system/jlib/jexcept.cpp

```diff
@@ -1284,6 +1286,7 @@ void  jlib_decl disableSEHtoExceptionMapping()
     signal(SIGBUS, SIG_DFL);
     signal(SIGILL, SIG_DFL);
     signal(SIGFPE, SIG_DFL);
+    signal(SIGABRT, SIG_DFL);
```

Comment:
I think an:
abort();
call is needed at the end of excsighandler() to generate a core file for when fireExecption() returns false.